models/subscriptions.go

```diff
@@ -11,17 +11,18 @@ import (
 )
 
 type Subscription struct {
-	ID   string `gorm:"unique;primary"`
-	Type string
+	ID        string `gorm:"unique;primary",json:"id"`
+	Type      string `json:"type"`
 
-	User     *User  `json:"user,omitempty"`
+  User     *User  `json:"user,omitempty"`
```

Comment:
Interesting - was a small attempt at how GitHub's new merge conflict tool works. Apparently it doesn't :P